common/services/opening-times.js

```diff
@@ -0,0 +1,56 @@
+// TODO capture opening hours in Prismic, then this can become part of prismic services
```

Comment:
Could we flow this as it's in `common`?
(And I was battling to see why we're calling `[].concat.apply` on `placesHoursArray` which typing would explain).